features/admin.users.v1/components/user-profile.scss

```
@@ -66,3 +66,6 @@
         flex-direction: row;
     }
 }
+.link.pointing {
```

Wrap this class with a parent class and use that parent class for the Alert component. This way we can scope this only to user profile sections alert since the "link pointing" is used in other components as well.

Ex:
```suggestion
.profile-locked-alert {
	.link.pointing {
	...
	}
}
```
`<Alert severity="warning" className="locked-alert">`

---

Resolved via https://github.com/wso2/identity-apps/pull/7992/commits/070b89ffe8036eac1f8ea66388f17d6f1226f175